tests/puppeteer/.docker/Dockerfile

```
@@ -22,4 +22,11 @@ WORKDIR /tests/
 COPY ["./", "./"]
 
 # Install dependencies
-RUN npm install
+RUN npm install \
+    && groupadd -r pptruser && useradd -r -g pptruser -G audio,video pptruser \
```

```suggestion
    && groupadd -r pptruser && useradd -r -g pptruser -m -G audio,video pptruser
```

And no need mkdir & chown.

Plus, I don't think you need to change the `USER`

---

I will test like that but this came from the official [DockerFile](https://github.com/puppeteer/puppeteer/blob/master/docs/troubleshooting.md#running-puppeteer-in-docker)

---

Oh, funny they used this kind of behaviour.
Ignore my comment :D Always try to use what they show you in the official documentation